app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestsFragment.java

```diff
@@ -24,6 +33,43 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState)
     {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_requests, container, false);
+        view = inflater.inflate(R.layout.fragment_requests, container, false);
+        current = view.findViewById(R.id.current);
+        current.setBackgroundColor(getResources().getColor(R.color.colorBackground));
+        toggle = view.findViewById(R.id.toggle_btn);
+        borrow = false;
+
+        toggle.setOnClickListener(this);
+
+        return view;
+    }
+
+    @Override
+    public void onClick(View view)
+    {
+        ConstraintLayout layout = this.view.findViewById(R.id.requests);
+        ConstraintSet cons = new ConstraintSet();
+        cons.clone(layout);
+
+        if(borrow) { // currently in borrow
+            current.setText(getResources().getString(R.string.my_requests));
+            toggle.setText((getResources().getString(R.string.borrow)));
+
+            cons.connect(R.id.toggle_btn, ConstraintSet.END, R.id.requests, ConstraintSet.END);
+            cons.connect(R.id.toggle_btn, ConstraintSet.START, R.id.current, ConstraintSet.END);
+            cons.connect(R.id.current, ConstraintSet.END, R.id.toggle_btn, ConstraintSet.START);
+            cons.connect(R.id.current, ConstraintSet.START, R.id.requests, ConstraintSet.START);
+        } else { // currently in my requests
+            current.setText(getResources().getString(R.string.borrow));
+            toggle.setText((getResources().getString(R.string.my_requests)));
+
+            cons.connect(R.id.current, ConstraintSet.END, R.id.requests, ConstraintSet.END);
+            cons.connect(R.id.current, ConstraintSet.START, R.id.toggle_btn, ConstraintSet.END);
```

Comment:
I think these should be normally camel case, can you check what the others are doing for buttons, etc?